### ISSUE TITLE
Undo and Redo icons are not consistent throughout the interface

### ISSUE THREAD

**Issue by karlgroves** [WordPress/gutenberg#15310]
Undo and Redo icons are not consistent throughout the interface

-   **Severity:** Low
-   **Affected Populations:**
    -   Cognitively Impaired
-   **Platform(s):**
    -   All / Universal
-   **Components affected:**
    -   Edit Media

#### Issue description

On the Edit Media page, there are various icon buttons for performing
actions to the selected media image, along with "Undo" and "Redo"
buttons.

These two buttons use different icons than the "Undo" and "Redo"
buttons in the Editor Top Bar.

Ideally these buttons' icons would be consistent with the ones in the
Editor Top Bar, especially since these icons look like Rotate icons
(causing the actual Rotate buttons to require more complex glyphs to
appear more clearly as Rotate-the-image), while the Editor Top Bar icons
do not look like Rotate buttons.

#### Remediation Guidance

Use the same icons for the "Undo" and "Redo" buttons as are used in
the Editor Top Bar.

#### Relevant standards

-   [3.2.4 Consistent Identification (Level
    AA)](https://www.w3.org/TR/WCAG20/#consistent-behavior-consistent-functionality)

**Note:** This issue may be a duplicate with other existing accessibility-related bugs in this project. This issue comes from the Gutenberg accessibility audit, performed by [Tenon](https://www.tenon.io) and funded by [WP Campus](https://wpcampus.org/). This issue is GUT-84 in Tenon's report

**Comment by afercia** [comment]
Worth noting the "Edit Media" page isn't strictly part of Gutenberg. It's part of core and accessible from other admin pages. Regardless, this poses an interesting point about design and UI consistency, which helps all users.

**Comment by kjellr** [comment]
We discussed this briefly in today's [Design triage meeting](https://wordpress.slack.com/archives/C02S78ZAL/p1556820168440200) (A WordPress.org slack login may be required to view that link).

Here are screenshots for reference: 

Editor Toolbar: 
<img width="330" alt="Screen Shot 2019-05-02 at 11 06 33 AM" src="https://user-images.githubusercontent.com/1202812/57096595-8ad63b00-6cca-11e9-8a07-26339b04ace7.png">

Media Modal: 
<img width="441" alt="Screen Shot 2019-05-02 at 11 07 28 AM" src="https://user-images.githubusercontent.com/1202812/57096597-8c076800-6cca-11e9-956f-c62a224fd059.png">

In general, we all definitely agree that these icons should be consistent. 

My personal preference would be that the media modal reuse the icons from the Editor Toolbar. As noted above, the media modal ones could be misinterpreted as rotate buttons.

**Comment by melchoyce** [comment]
For reference, these icons also appear in Attachment pages:

![image](https://user-images.githubusercontent.com/2846578/57097048-84948e80-6ccb-11e9-831b-d660b20285cd.png)

We'll want to update everywhere.

**Comment by afercia** [comment]
From a technical perspective, this implies introducing SVG icons in core, which uses only Dashicons so far. I guess there's large consensus about consistency 🙂

**Comment by afercia** [comment]
Moved to Trac: https://core.trac.wordpress.org/ticket/47118
